addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2107,8 +2107,14 @@ body.editor_enable.editor_has_snippets {
                     min-width: 22px;
                     padding: 0 $o-we-sidebar-content-field-button-group-button-spacing * .5;
                     color: $o-we-fg-lighter;
-                    &.oe_snippet_remove {
+                    // TODO In master: restore `.oe_snippet_remove`.
+                    &.fa-trash {
                         background-color: mix($o-we-color-danger, $o-we-sidebar-content-field-clickable-bg);
+                        margin-left: $o-we-sidebar-content-field-button-group-button-spacing;
```

Comment:
Let's not add that, it's changing the layout, it was probably removed for a reason in 15.0